x-pack/solutions/observability/plugins/infra/public/components/asset_details/components/section_title.tsx

```diff
@@ -28,19 +28,21 @@ export const TitleWithTooltip = ({
   'data-test-subj': dataTestSubject,
   tooltipTestSubj,
   tooltipContent,
+  tooltipAriaLabel,
 }: {
   title: string;
   tooltipContent: ReactNode;
   'data-test-subj'?: string;
   tooltipTestSubj?: string;
+  tooltipAriaLabel?: string;
 }) => {
   return (
     <EuiFlexGroup gutterSize="xs" alignItems="center" responsive={false}>
       <EuiFlexItem grow={false}>
         <SectionTitle title={title} data-test-subj={dataTestSubject} />
       </EuiFlexItem>
       <EuiFlexItem grow={false}>
-        <Popover icon="iInCircle" data-test-subj={tooltipTestSubj}>
+        <Popover icon="iInCircle" data-test-subj={tooltipTestSubj} aria-label={tooltipAriaLabel}>
```

Comment:
We could always default the `aria-label` to the **title** where they have the same value like here:

- x-pack/solutions/observability/plugins/infra/public/components/asset_details/charts/docker_charts.tsx
- x-pack/solutions/observability/plugins/infra/public/components/asset_details/charts/host_charts.tsx
- x-pack/solutions/observability/plugins/infra/public/components/asset_details/charts/kubernetes_charts.tsx

Reply:
change it here https://github.com/elastic/kibana/pull/218766/commits/44c4b8c564b49df4ff2e1db0944e29b292ee056f